client/src/components/Sidebar/Sidebar.module.scss

```diff
@@ -1,9 +1,13 @@
 $width: 215px;
 
 .root {
+    overflow-y: auto;
+    flex-wrap: nowrap;
```

Comment:
Prevent nav links from wrapping to a second column on a narrow screen. This is so easy a solution to so rare a problem, that is is not documented in "Testing" steps for this PR.